packages/ffmpeg.rb

```diff
@@ -204,10 +204,9 @@ def self.build
         --enable-shared \
         --enable-version3 \
         #{@mfx}  \
-        --host-cflags='-pipe -fno-stack-protector -U_FORTIFY_SOURCE #{@lto} -fuse-ld=#{CREW_LINKER} #{@arch_cflags}' \
-        --host-ldflags='-fno-stack-protector -U_FORTIFY_SOURCE #{@lto} #{@arch_cflags}' \
-        #{CREW_OPTIONS.sub(/--build=.*/, '').gsub('vfpv3-d16', 'neon')}"
-
+        --host-cflags='#{CREW_ENV_OPTIONS_HASH['CFLAGS']} #{@lto} -fuse-ld=#{CREW_LINKER} #{@arch_cflags}' \
+        --host-ldflags='#{CREW_ENV_OPTIONS_HASH['LDFLAGS']} #{@lto}' \
+        #{CREW_OPTIONS.sub(/--build=.*/, '').gsub('vfpv3-d16', 'neon').gsub('--disable-dependency-tracking', '')}"
```

Comment:
Why are we removing `--disable-dependency-tracking`?

Reply:
The configure script complains about it. Not all configure scripts can handle it.